src/data/complex/exponential.lean

```diff
@@ -223,6 +223,14 @@ begin
     (λ _ _, rfl),
 end
 
+lemma sum_range_add_sub_sum_range {α : Type*} [add_comm_group α] {f : ℕ → α}
+  {n m : ℕ} : ∑ k in range (n + m), f k - ∑ k in range n, f k =
```

Comment:
```suggestion
lemma sum_range_add_sub_sum_range {α : Type*} [add_comm_group α] (f : ℕ → α)
  (n m : ℕ) : ∑ k in range (n + m), f k - ∑ k in range n, f k =
```

Reply:
I'd also suggest moving this lemma to just below these lines
https://github.com/leanprover-community/mathlib/blob/a983f2411b886c3f1b011738c16777f49003e508/src/algebra/big_operators/basic.lean#L769-L777
and calling it `prod_range_add_div_prod_range` so that `to_additive` can generate your lemma.

Reply:
Are you sure this is possible? None of the other lemmas in that file use `div` and when I try to do this, it complains that all the lemmas with div are for `comm_group_with_zero` rather than `comm_group` used throughout the file

Reply:
Perhaps it should go in the `comm_group_with_zero` section? But the only other lemma there is not `to_additive`

Reply:
Just add a local `[comm_group β]` argument

Reply:
What I'm saying is, when I do that, the following library search fails:
```
lemma foo {α : Type*} [comm_group α]
  (a b c : α) : a / b = c -> a = c * b :=
begin
  -- library_search, -- fails
end
```
In my mind, this is a pretty basic theorem about groups so it should be in the library somewhere, and because `\alpha` has instance `comm_group`, library search should find it. What is the proper lemma and where is it?

Reply:
It seems like group.basic just writes all these in terms of inverses, so using `div_eq_mul_inv` and then some tinkering works. Still feels suboptimal though.

Reply:
Are you saying that you can't find a multiplicative version `sub_eq_of_eq_add'`?

Reply:
All I can find is https://leanprover-community.github.io/mathlib_docs/algebra/group_with_zero/basic.html#div_eq_of_eq_mul, which is for `group_with_zero`

Reply:
Let me continue this conversation in the PR thread on Zulip